### DISCUSSION TITLE
How to limit result returns when a user enters a sole  *" an asterisk into the search field?

### DISCUSSION THREAD

**Discussion by tedtninety** [microsoft-search/pnp-modern-search#2766] [Q&A] [answered]
I have two search terms in the query template.
Two strings with an OR statement.
I tried  (RefinableString123:({SearchTerms}) OR Refinabl;String321(:{SearchTerms})  AND (path:https:/domain/sites/this)).
However, I still can't manage to lock down the results when a sole asterisk is used in the search.
Any help would be appreciated.

**Comment by kasperbolarsen** [comment, marked as answer]
Hi @tedtninety 

Could you explain what you are trying to achieve.  Is it something like "any item that fits the specified path and where the two refinablestrings contains a value,  aka not null"?


those two refinablestrings, what are they mapped to, metadata fields  perhaps?

**Reply by tedtninety** [reply]
Hi @kasperbolarsen, thank you for your reply.

I have a list that is looked up via PnP Search. The keyword looks through both the aforementioned RefinableStrings ( Name and Description  columns within the same list , respectively ) for the keyword match; it works fine.
However, when an * is entered as a keyword , it brings up a ton of results that have nothing to do with either RefinableStrings or the list they belong to.

I thought perhaps adding the path name to the list as an "AND" would remedy that , but to no avail.

How do I address this so that wither:
- the asterisk displays nothing OR
-  It only displays results form the given list?

Thank you,
ted

**Reply by kasperbolarsen** [reply]
I tried to see if I could see the same issue as you. I can't.

this query in the Query template works just as expected with *:

(refinableString01:{searchTerms} OR RefinableString02:{searchTerms}) Path:"https://tcwlv.sharepoint.com/sites/PnPModernSearch/Lists/Datalist*" ContenttypeId:"0x01008305385FA140D747A5C15AA6E00590B900214A6099A8B3594189A662A74BC65E9B*"

**Reply by tedtninety** [reply]
Hi @kasperbolarsen . I will try that for sure and let you know.
I'd like to verify that the ContentTypeId is from the path - list?
Thank you so very much!, I am so grateful :)

**Reply by tedtninety** [reply]
@kasperbolarsen  - YOU Are Brilliant. Thank you for teaching me how to fish! :)
I did not know there was a need to append an * to the path itself! 
Thank you so much!

BTW, if you are ever curious to recreate the issue, removing the path does it.

**Reply by wobba** [reply]
Adding a * should usually not be needed (due to path being a bit special).

**Reply by tedtninety** [reply]
@wobba  Thank you for the clarification.
